src/main/java/uk/gov/hmcts/cmc/claimstore/models/CountyCourtJudgment.java

```diff
@@ -0,0 +1,108 @@
+package uk.gov.hmcts.cmc.claimstore.models;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+import uk.gov.hmcts.cmc.claimstore.models.ccj.PaymentOption;
+import uk.gov.hmcts.cmc.claimstore.models.ccj.RepaymentPlan;
+import uk.gov.hmcts.cmc.claimstore.models.legalrep.StatementOfTruth;
+import uk.gov.hmcts.cmc.claimstore.models.otherparty.TheirDetails;
+
+import java.time.LocalDate;
+import java.util.Objects;
+import java.util.Optional;
+import javax.validation.Valid;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+import static uk.gov.hmcts.cmc.claimstore.utils.ToStringStyle.ourStyle;
+
+@JsonInclude(JsonInclude.Include.NON_ABSENT)
+public class CountyCourtJudgment {
+
+    @Valid
+    @NotNull
+    private final TheirDetails defendant;
+
+    @Min(0)
+    private double paidAmount;
+
+    @Valid
+    @NotNull
+    private final PaymentOption paymentOption;
+
+    @Valid
+    private final RepaymentPlan repaymentPlan;
+
+    @Valid
+    private final LocalDate payBySetDate;
+
+    @Valid
+    private final StatementOfTruth statementOfTruth;
+
+    public CountyCourtJudgment(
+        TheirDetails defendant,
+        PaymentOption paymentOption,
+        Double paidAmount,
+        RepaymentPlan repaymentPlan,
+        LocalDate payBySetDate,
+        StatementOfTruth statementOfTruth) {
```

Comment:
move the closing bracket down a line?
(means less diff if you add a new field)

Reply:
ok